test/unit/factories/Container.mocha.ts

```diff
@@ -0,0 +1,69 @@
+/// <reference path='../test.mocha.ts' />
+
+describe('n3Charts.Factory.Container', () => {
+  var domElement: JQuery = undefined;
+  var container: n3Charts.Factory.Container = undefined;
+
+  beforeEach(() => {
+    domElement = angular.element(document.body).append('<div></div>');
```

Comment:
this appends one element per tests. Maybe it would be better to append once, and then truncate it before every test